.gitignore

```diff
@@ -15,11 +15,13 @@ coverage.xml
 # Dependencies
 !requirements.txt
 
-# Sphinx documentation
-/doc/_build/
-/doc/_theme/
-/doc/pyEDAA.UCIS/**/*.*
-!/doc/pyEDAA.UCIS/index.rst
+# Sphinx
+doc/_build/
+doc/pyEDAA.UCIS/**/*.*
+!doc/pyEDAA.UCIS/index.rst
 
-# PyCharm project
+# BuildTheDocs
+doc/_theme/**/*.*
```

Comment:
It's not specfic about BuildTheDocs. It could be used for any local theme. I would keep it in the Sphinx section, since `_theme` is used in `doc/conf.py`.

Reply:
Fixed in e29730d7.